### PR TITLE
refactor(client): replace text with internationalizing formatter calls. close #242

### LINKED ISSUE #242: Replace text in the client app with internationalizing formatter calls to expedite translation
### Background on the problem the feature will solve/improved user experience
When users come to the application we want them to be able to read text in the language they're most comfortable. In order to do this we must replace all plain text displays with calls to the internationalization formatter. At the time of this issue being created that is `react-intl`. You may see examples in the client app by looking for `intl.formatMessage`

### Describe the solution you'd like
Display text should be moved to the `/src/nls/messages/en/messages.js` file and then properly looked up from there by the component.

### Tasks 
For each piece of text being moved
- [ ] Write a component test (or add an assertion to an existing test) to verify the text displays on the screen
- [ ] Move the text to the English translation file. Do not add to other translation files unless you can also provide that translation. Hint: Reuse things like 'Add', 'Update', etc...
- [ ] Use the formatting pattern in place to pull the text
- [ ] Make sure test still passes

### Acceptance Criteria 
All of the remaining display text (text the user sees) has been moved to the English translation file.
Tests exist for the items moved.

### PR DESCRIPTION
**Pull request by Fiewor**
Contributes to #242 

## What did you do?
Replace plain text with internationalizing formatter calls to expedite translation

## Why did you do it?
To enable users to view text in their preferred (available) language

## How have you tested it?
No

## Were docs updated if needed?
- [x] No
- [ ] Yes
- [ ] N/A

#### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new console logs
- [x] Fixes entire issue
- [ ] Partial fix for issue

### CODE REVIEW

**Comment by ed-snodgrass** [comment]
Hey @Fiewor , just a heads up that it failed to build this because you've checked in code using CR line endings instead of LF. 

Depending on what IDE you use there ought to be information that you can search for to show you how to adjust that.

**Comment by ed-snodgrass** [comment]
It looks like RecordRoute and PolicyRoute has some strings that could be updated. I think your change was correct so once the line endings are fixed it should be a quick approval/merge. 

Thanks @Fiewor !

**Comment by Fiewor** [comment]
> Hey @Fiewor , just a heads up that it failed to build this because you've checked in code using CR line endings instead of LF.
> 
> Depending on what IDE you use there ought to be information that you can search for to show you how to adjust that.

Oh. I'm just seeing that I missed this @ed-snodgrass . I'm using Visual Studio Code and yes the line ending is CRLF. I'll fix that before pushing.

**Comment by Fiewor** [comment]
So, somehow, running `git status` shows that these files have been modified. 
Apart from my changes, all I did was run `npm run lint:fix`
I don't think I should be pushing all these with these PR, but could it be the reason the build is failing? @ed-snodgrass 

image

**Review by ed-snodgrass — CHANGES_REQUESTED** [review]
@Fiewor Thank you for this. Exactly what was needed. 

I made one comment where "Blur" is used and there is another spot around line 41 in PolicyTable.

I'm approving the changes in case

Thanks again.

File: client/src/policyRoute/PolicyRoute.js [hunk 1]
```diff
@@ -25,11 +26,23 @@ const PolicyRoute = () => {
...
   }
-  return <h2 data-testid={'emptyPolicy'}>No policy found</h2>
+  return (
+    <h2 data-testid={'emptyPolicy'}>
+      {intl.formatMessage(messages.noPolicyFound)}
+    </h2>
+  )
 }
 
 export default PolicyRoute
```
[the remark this answers is not in the document]
Comment (Fiewor) [reply client/src/policyRoute/PolicyRoute.js:48]: Thank you Ed. I'll make the changes and push

**Comment by Fiewor** [comment]
Hi @ed-snodgrass .Thanks for all the help. 
I just read the docs for react-intl and I saw that there's a <FormattedMessage /> and <FormattedNumber /> that I think would be helpful and also a 'description' attribute which should help accessibility too.
What do you think? Should I try applying these? Or you'll make it a seperate issue?

File: client/src/policyRoute/PolicyRoute.js [hunk 2]
```diff
@@ -25,11 +27,27 @@ const PolicyRoute = () => {
...
   }
-  return <h2 data-testid={'emptyPolicy'}>No policy found</h2>
+  return (
+    <h2 data-testid={'emptyPolicy'}>
+      {intl.formatMessage(messages.noPolicyFound)}
+    </h2>
+  )
+}
+
+PolicyRoute.propTypes = {
```
Comment (ed-snodgrass) [inline client/src/policyRoute/PolicyRoute.js:49]: you're missing the import for PropTypes

**Comment by ed-snodgrass** [comment]
> Hi @ed-snodgrass .Thanks for all the help. I just read the docs for _react-intl_ and I saw that there's a `<FormattedMessage />` and `<FormattedNumber />` that I think would be helpful and also a `description` attribute which should help accessibility too. What do you think? Should I try applying these? Or maybe you could make it a separate issue?

It is probably worth looking into but as a separate issue. Once this PR is merged you might want to play around with `<FormattedMessage />` and `<FormattedNumber />` and see if you think it is something we should look into.

Thanks again.

File: client/src/policyRoute/PolicyRoute.js (hunk 2 above)
Comment (Fiewor) [reply client/src/policyRoute/PolicyRoute.js:49]: My bad. I'll resolve this

**Comment by Fiewor** [comment]
> > Hi @ed-snodgrass .Thanks for all the help. I just read the docs for _react-intl_ and I saw that there's a `<FormattedMessage />` and `<FormattedNumber />` that I think would be helpful and also a `description` attribute which should help accessibility too. What do you think? Should I try applying these? Or maybe you could make it a separate issue?
> 
> It is probably worth looking into but as a separate issue. Once this PR is merged you might want to play around with `<FormattedMessage />` and `<FormattedNumber />` and see if you think it is something we should look into.
> 
> Thanks again.

Alright!

File: client/src/policyRoute/PolicyRoute.js [hunk 3]
```diff
@@ -3,6 +3,8 @@ import { useParams } from 'react-router-dom'
 import { useDispatch, useSelector } from 'react-redux'
 import { fetchCurrentPolicy } from '../store/policy.duck'
 import PolicyDetail from '../policyDetail/PolicyDetail'
+import { messages } from '../nls/nlsUtility'
+import { injectIntl } from 'react-intl'
 
 const PolicyRoute = () => {
```
[the remark this answers is not in the document]
Comment (Fiewor) [reply client/src/policyRoute/PolicyRoute.js:10]: Thank you @ed-snodgrass 
Apologies for not completing this at once and having to correct errors multiple times

**Review by ed-snodgrass — APPROVED** [review]
Thanks again @Fiewor . 

This fixes #242

**Pull request merged**